modules/host/consts.go

```diff
@@ -104,6 +131,26 @@ var (
 	// data.
 	defaultUploadBandwidthPrice = types.SiacoinPrecision.Mul64(10).Div(modules.BytesPerTerabyte) // 10 SC / TB
 
+	// workingStateFrequency defines how frequently the Host's working status
+	// check runs
+	workingStateFrequency = build.Select(build.Var{
+		Standard: time.Minute * 15,
+		Dev:      time.Minute * 5,
+		Testing:  time.Second * 30,
+	}).(time.Duration)
+
+	// workingStateThreshold defines how many settings calls must occur over the
+	// workingStateFrequency for the host to be considered working.
+	workingStateThreshold = uint64(3)
+
+	// connectablityCheckFrequency defines how often the host's connectability
+	// check is run.
+	connectabilityCheckFrequency = build.Select(build.Var{
+		Standard: time.Minute * 3,
+		Dev:      time.Minute,
```

Comment:
can probably put the values here higher. Maybe 10 minutes for Standard and 3 minutes for Dev?